### PR TITLE
Added Session report for Constraint Layout Session 1.md 2020 batch

### PR DESCRIPTION
**Pull request by 0mKumar**
Added first session (Constraint Layout) report

### CODE REVIEW

File: Android Development/2020-2021/Session 1.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<!-- 
+ - Replace all the {} with their values. Name this file as Session {Session no.}.md (without braces) and submit a PR.
+-->
+# Session 1
+Introduction to Constraint Layout
```
Comment (rishabh-997) [inline Android Development/2020-2021/Session 1.md]: change it to ConstraintLayout

File: Android Development/2020-2021/Session 1.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+Conducted on: 27.01.2020
+
+## Agenda
+An Introduction to Constraint Layout
+
+## Summary
+Constraint Layout is better performance wise because it avoids helps view hierarchy. Flat layouts are quicker to render.
+Various constraints available to create flat complex layouts using Constraint Layout was taught.
+
+Disscussion included:
```
Comment (rishabh-997) [inline Android Development/2020-2021/Session 1.md]: check spelling

Comment (rishabh-997) [reply Android Development/2020-2021/Session 1.md]: cross check through grammarly too...

File: Android Development/2020-2021/Session 1.md (hunk 1 above)
Comment (0mKumar) [reply Android Development/2020-2021/Session 1.md]: Okay, done

File: Android Development/2020-2021/Session 1.md (hunk 2 above)
Comment (0mKumar) [reply Android Development/2020-2021/Session 1.md]: Thanks for pointing it out, corrected spelling errors.

**Comment by rishabh-997** [comment]
@ShridharGoel sir, merge it please

**Comment by rishabh-997** [comment]
@0mKumar , update the location of file to [this](https://github.com/Cyber-Labs/Session-Reports/tree/master/Android%20Development/2019-20)

**Pull request merged**